python-sdk/tests/benchmark/results.md

```diff
@@ -248,3 +248,54 @@ Note - These results are generated manually, there is a issue added for the same
 | bigquery   | one_gb    | 12.23min     | 231.33MB     | 10.5min         | 1.15min           |
 | bigquery   | ten_kb    | 17.06s       | 61.65MB      | 7.16s           | 1.01s             |
 | bigquery   | ten_mb    | 15.22s       | 66.08MB      | 7.35s           | 940.0ms           |
+
+
+# Performance evaluation of loading datasets from AWS S3 with Astro Python SDK 1.1.0 into Redshift
+
+### Without native support
+
+Astro Python SDK 1.1.0 supports loading to Redshift using Pandas, without any further optimizations.
+
+The benchmark was run as a Kubernetes job in GKE:
+
+* Version: `astro-sdk-python` 1.1.0
+* Machine type: `n2-standard-4`
+  * vCPU: 4
+  * Memory: 16 GB RAM
+* Container resource limit:
+  * Memory: 10 Gi
+
+| database   | dataset    | total_time   | memory_rss   | cpu_time_user   | cpu_time_system|
+|:-----------|:-----------|:-------------|:-------------|:----------------|:---------------|
+| redshift   | ten_kb     | 21.1s        | 59.57MB      | 910.0ms         | 100.0ms        |
+| redshift   | hundred_kb | 1.15min      | 59.22MB      | 1.16s           | 180.0ms        |
+| redshift   | one_mb     | 11.01min     | 62.89MB      | 3.46s           | 810.0ms        |
+
+The `pandas.to_sql` function uses highly suboptimal INSERT statement for Redshift. Hence, you're not recommended
+to use this approach for large files, but instead use the native approach.
```

Comment:
Let's have a look at the current pandas implementation tomorrow and how we can improve it with batch `INSERT`ion, before publishing them. WDYT?

Reply:
The release is to be cut tonight. I will try the approach that you suggested later tonight, but it looks a bit unclean to me. We can also connect tomorrow. Just wanted to draft the findings as per the current implementation if a beta has to be cut today.